src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -97,6 +97,8 @@ public static string iOSBrokerKeySaveFailed(string keyChainResult)
             +
             "2. An error occurred during a silent web authentication that prevented the http authentication flow from completing in a short enough time frame";
 
+        public const string StateMismatchErrorMessage = "Returned state({0}) from authorize endpoint is not the same as the one sent({1}). See https://aka.ms/msal-statemismatcherror for more details.";
```

Comment:
Is it good from a security point of view to display the state that was sent?

Reply:
I don't think it would ,matter as it has already failed. @bgavrilMS Suggestions?

Reply:
that's a good point, plus what would you do w/those values? and they are pretty long. i think just having the error message w/the link is enough.

Reply:
@jennyf19 Those were already there in the message. I just added the aka.ms link and moved the message to this class. I don't know what they were added for.